src/servers/sapphire_zone/Network/Handlers/ClientTriggerHandler.cpp

```diff
@@ -118,6 +120,40 @@ void Core::Network::GameConnection::clientTriggerHandler( const Packets::FFXIVAR
         player.getCurrentAction()->setInterrupted();
       break;
     }
+    case ClientTriggerType::Examine:
```

Comment:
Might wanna start putting big blocks like this into extra functions.

Reply:
i was considering creating an examine packet class, yea